baseapp/msg_service_router.go

```diff
@@ -92,3 +88,10 @@ func (msr *MsgServiceRouter) RegisterService(sd *grpc.ServiceDesc, handler inter
 func (msr *MsgServiceRouter) SetInterfaceRegistry(interfaceRegistry codectypes.InterfaceRegistry) {
 	msr.interfaceRegistry = interfaceRegistry
 }
+
+// gRPC NOOP interceptor
+func noopInterceptor(_ context.Context, _ interface{}, _ *grpc.UnaryServerInfo, _ grpc.UnaryHandler) (interface{}, error) {
```

Comment:
```suggestion
func noopInterceptor(_ interface{}, _ interface{}, _ interface{}, _ interface{}) (interface{}, error) {
```

Reply:
I think this would work too

Reply:
true, though I prefer seeing the exact signature from gogogrpc

Reply:
exactly, no need to generalize here.